### PR TITLE
Added another game

### PR DESCRIPTION
**Pull request by VMistry**
I have done a lot in this one...
- added another page to the website
- Created nav bar
- Linked the two pages.

### CODE REVIEW

File: Game1/Game1.html [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+    <!-- Latest compiled and minified CSS -->
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
+
+    <!-- Optional theme -->
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap-theme.min.css" integrity="sha384-rHyoN1iRsVXV4nD0JutlnGaslCJuC7uwjduW9SVrLvRYooPp2bWYgmgJQIXwl/Sp" crossorigin="anonymous">
+
+    <!-- Latest compiled and minified JavaScript -->
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js" integrity="sha384-Tc5IQib027qvyjSMfHjOMaLkfuWVxZxUPnCJA7l2mCWNIpG9mGCD8wGNIcPD7Txa" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="style.css">
```
Comment (AEJackson) [inline Game1/Game1.html:14]: are you including your style sheet twice?

**Review by AEJackson — CHANGES_REQUESTED** [review]

File: Game1/Game1.html (hunk 1 above)
Comment (richgurney) [reply Game1/Game1.html:14]: Game 1 - Isn't working for me lets take a look in the morning

**Comment by VMistry** [comment]
I have changed the CSS(style.css) and Javascript(script.js) files to CSS(game2Style.css) and Javascript(game2Script.js) in the Game2 folder.

File: Game1/script.js [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+var red = $(".innerSquare", "#red").css('background-color', 'red').hide();
+var blue = $(".innerSquare", "#blue").css('background-color', 'blue').hide();
+var green = $(".innerSquare", "#green").css('background-color', 'green').hide();
+var yellow = $(".innerSquare", "#yellow").css('background-color', 'yellow').hide();
+
+//Start the game by creating the sequence.
+$("#patternSet").on("click", createLevel);
+
+//This is used to create the squence for the level.
+async function createLevel(){
```
Comment (richgurney) [inline Game1/script.js:25]: What does this mean?

File: Game1/script.js [hunk 3]
```diff
@@ -0,0 +1,181 @@
...
+        player++;
+        //Display message saying its player ones turn.
+        $('#updatingTXT1').text("Player " + player + "'s turn");
+      }
+    }
+    $('#updatingTXT3').text("Ready to Process");
+  }
+}
+
+//compare the computer generated array with the array created by player.
```
Comment (richgurney) [inline Game1/script.js:130]: Nice comments

**Review by richgurney — APPROVED** [review]

**Review by AEJackson — APPROVED** [review]

**Pull request merged**